examples/geoObject/geoObject.js

```diff
@@ -0,0 +1,185 @@
+            };
+        }
+    };
+
+const div = document.createElement('div');
+div.style.setProperty('display', 'flex');
+div.style.setProperty('flex-direction', 'column');
+div.style.setProperty('position', 'absolute');
+div.style.setProperty('top', '10px');
+div.style.setProperty('left', '10px');
+div.style.setProperty('color', 'white');
+div.style.setProperty('background', `rgba(0, 0, 0, .7)`);
+document.body.appendChild(div);
+
+const span = document.createElement('span');
+span.setAttribute('id', 'instance-count');
+span.innerText = `Instance count: ${COUNT}`;
+div.appendChild(span);
+
+const range = document.createElement('input');
+range.setAttribute('type', 'range');
+range.setAttribute('min', '1');
+range.setAttribute('value', '1');
+range.setAttribute('max', '5000');
+range.addEventListener('input', () => {
+    COUNT = parseInt(range.value);
+});
+div.appendChild(range);
+
+function rnd(min, max) {
+    return Math.random() * (max - min) + min;
+}
+
+let globus = new Globe({
+    target: "globus",
+    name: "Earth",
+    layers: [
+        new XYZ("OpenStreetMap", {
+            isBaseLayer: true,
+            url: "//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+            visibility: true,
+            attribution: "Data @ OpenStreetMap contributors, ODbL"
+        })
+    ],
+    terrain: new GlobusTerrain()
+});
+let colors = ["red", "orange", "yellow", "green", "lightblue", "darkblue", "purple"];
+
+let geoObjects = new EntityCollection({
+    entities: [],
+    scaleByDistance: [600000, 24000000, 10000000000]
+});
+
+const instancesTypes = [...new Array(80).fill('airplane'), ...new Array(20).fill('farmplane'), 'satellite'];
+
+const addInstance = (name, options = {}) => {
+    fetch(`./${name}.json`)
+        .then((response) => {
+            return response.json();
+        })
+        .then((data) => {
+            const entities = [];
+            const { vertices, indices, normals } = data,
+                options = (i) => ({
+                    name: "sat-" + i,
+                    geoObject: {
+                        scale: 100000,
+                        instanced: true,
+                        tag: name,
+                        color: colors[i % 7],
+                        vertices,
+                        indices,
+                        normals
+                    },
+                    'properties': {
+                        'color': colors[i % 7]
+                    }
+                });
+            ENTITY[name] = (i) => {
+                const o = options(i);
+                return {
+                    ...o,
+                    ...(ENTITY_DEFAULT_OPTIONS[name] ? ENTITY_DEFAULT_OPTIONS[name](o, i) : {})
+                };
+            };
+
+            for (let i = 0; i < COUNT; i++) {
+                entities.push(new Entity(ENTITY[name](i)));
+            }
+            geoObjects.addEntities(entities);
+        });
+};
+
+instancesTypes.forEach((type) => {
+    addInstance(type);
```

Comment:
In terms of my previous comment, I m afraid that this makes a lot of fetches, no?